.gitmodules

```diff
@@ -1,39 +1,82 @@
-[submodule "vendor/blip-buf"]
-	path = vendor/blip-buf
-	url = https://github.com/nesbox/blip-buf.git
-[submodule "vendor/curl"]
-	path = vendor/curl
-	url = https://github.com/nesbox/curl.git
-[submodule "vendor/dirent"]
-	path = vendor/dirent
-	url = https://github.com/tronkko/dirent.git
-[submodule "vendor/duktape"]
-	path = vendor/duktape
-	url = https://github.com/svaarala/duktape-releases.git
-[submodule "vendor/giflib"]
-	path = vendor/giflib
-	url = https://github.com/nesbox/giflib.git
-[submodule "vendor/lpeg"]
-	path = vendor/lpeg
-	url = https://github.com/nesbox/lpeg.git
-[submodule "vendor/lua"]
-	path = vendor/lua
-	url = https://github.com/lua/lua.git
-[submodule "vendor/sdl-gpu"]
-	path = vendor/sdl-gpu
-	url = https://github.com/grimfang4/sdl-gpu.git
+[submodule "vendor/blip-buf"]
+	path = vendor/blip-buf
+	url = https://github.com/nesbox/blip-buf.git
+	shallow = true
+[submodule "vendor/curl"]
+	path = vendor/curl
+	url = https://github.com/curl/curl.git
+	shallow = true

```

Comment:
Looks like a lot of changes are showing up in this diff that have already been applied. Did something happen in the rebase that resulted in a lot of conflicts? Wouldn't want to lose any code changes.

Reply:
https://github.com/nesbox/TIC-80/pull/1006#discussion_r765021911
> Nah, it's just that GitHub is broken since I merged master.
>
> It's not just GitHub doing that, I have seen that very behavior on GitLab @work too. Maybe I'm doing something wrong? :confused:

Reply:
You can see the relevant changes here: https://github.com/nesbox/TIC-80/compare/master...remi6397:feature/ruby/mruby.patch